src/asciidoc/docs/blueprints/customVCC.adoc

```diff
@@ -233,8 +232,8 @@ You should see your custom VCC.
 
 Custom VCCs require a custom domain to be accessible to other Koji applications.
 
-. Click *Publish now* and enter a *Name*, *Description* and other publish settings, as desired.
-Then, click *Publish*.
+. Click *Publish Now* and enter a *Name*, *Description* and other publish settings, as desired.
+Then, click *Publish New Version*.
 . When publishing is completed, click the link to open your live template on Koji.
 . Click the *Koji* button in the top right, and then go to *Koji Settings > Custom Domains*.
```

Comment:
The "Koji Settings" button has changed, too. Can you update? Worth another search of the repo for this term, too